specification/nodes/nodes_info/NodeInfo.ts

```diff
@@ -34,22 +36,226 @@ import { NodeThreadPoolInfo } from './NodeThreadPoolInfo'
 export class NodeInfo {
   attributes: Dictionary<string, string>
   build_flavor: string
+  /** Short hash of the last git commit in this release. */
   build_hash: string
   build_type: string
+  /** The node’s host name. */
   host: Host
-  http: NodeInfoHttp
+  http?: NodeInfoHttp
+  /** The node’s IP address. */
   ip: Ip
-  jvm: NodeJvmInfo
+  jvm?: NodeJvmInfo
+  /** The node's name */
   name: Name
-  network: NodeInfoNetwork
-  os: NodeOperatingSystemInfo
-  plugins: PluginStats[]
-  process: NodeProcessInfo
+  network?: NodeInfoNetwork
+  os?: NodeOperatingSystemInfo
+  plugins?: PluginStats[]
+  process?: NodeProcessInfo
   roles: NodeRole[]
-  settings: string[]
-  thread_pool: Dictionary<string, NodeThreadPoolInfo>
-  total_indexing_buffer: long
-  transport: NodeInfoTransport
+  settings?: NodeInfoSettings
+  thread_pool?: Dictionary<string, NodeThreadPoolInfo>
+  /**
+   * Total heap allowed to be used to hold recently indexed documents before they must be written to disk. This size is a shared pool across all shards on this node, and is controlled by Indexing Buffer settings.
+   * @doc_url https://www.elastic.co/guide/en/elasticsearch/reference/master/indexing-buffer.html
+   */
+  total_indexing_buffer?: long
+  /** Same as total_indexing_buffer, but expressed in bytes. */
+  total_indexing_buffer_in_bytes?: ByteSize
+  transport?: NodeInfoTransport
+  /** Host and port where transport HTTP connections are accepted. */
   transport_address: TransportAddress
+  /** Elasticsearch version running on this node. */
   version: VersionString
+  modules?: PluginStats[]
+  ingest?: NodeInfoIngest
+  aggregations?: Dictionary<string, NodeInfoAggregation>
+}
+
+export class NodeInfoSettings {
+  cluster: NodeInfoSettingsCluster
+  node: NodeInfoSettingsNode
+  path: NodeInfoPath
+  repositories?: NodeInfoRepositories
+  discovery?: NodeInfoDiscover
+  action?: NodeInfoAction
+  client: NodeInfoClient
+  http: NodeInfoSettingsHttp
+  bootstrap?: NodeInfoBootstrap
+  transport: NodeInfoSettingsTransport
+  network?: NodeInfoSettingsNetwork
+  xpack?: NodeInfoXpack
+  script?: NodeInfoScript
+  search?: NodeInfoSearch
+}
+
+export class NodeInfoSettingsCluster {
+  // @aliases cluster.name */
```

Comment:
This is not a valid JSDoc comment :)

```suggestion
  /** @aliases cluster.name */
```

Same below.